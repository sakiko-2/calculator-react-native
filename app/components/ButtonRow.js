import React from 'react';
import { View, StyleSheet } from 'react-native';

const ButtonRow = ({ children }) => {
  return (
    <View style={styles.buttonRow}>
      {children}
    </View>
  );
};

const styles = StyleSheet.create({
  buttonRow: {
    flex: 1,
    flexDirection: 'row',
    justifyContent: 'space-between'
  }
});

export default ButtonRow;