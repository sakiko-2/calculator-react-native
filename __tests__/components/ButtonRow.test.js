import 'react-native';
import React from 'react';
import ButtonRow from '../../app/components/ButtonRow';

import renderer from 'react-test-renderer';

describe('ButtonRow', () => {
  it ('renders correctly', () => {
    const tree = renderer.create(<ButtonRow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
