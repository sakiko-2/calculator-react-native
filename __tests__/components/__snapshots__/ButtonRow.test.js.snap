// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonRow renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
/>
`;
